controller/api/public/prometheus.go

```diff
@@ -46,6 +47,11 @@ func extractSampleValue(sample *model.Sample) uint64 {
 func (s *grpcServer) queryProm(ctx context.Context, query string) (model.Vector, error) {
 	log.Debugf("Query request:\n\t%+v", query)
 
+	_, span := trace.StartSpan(ctx, "query.prometheus")
+	defer span.End()
+	span.Annotate([]trace.Attribute{
```

Comment:
This ends up showing as a log event in Jaeger rather than a tag.  Not a big deal either way, but it's a bit surprising.  I would definitely have expected this data to be a tag.

Reply:
Moved it as a tag